test/FieldVector.jl

```diff
@@ -18,19 +18,21 @@
         @test length(p) === 3
         @test eltype(p) === Float64
 
+        @test (p + p)::Point3D ≈ Point3D(2.0, 4.0, 6.0)
```

Comment:
Can use `===` here, I think, to check that both the elements and type are the same.  (Elements should sum exactly, as the result is exactly representable in `Float64`.)